gwpy/timeseries/timeseries.py

```diff
@@ -1428,14 +1428,17 @@ def taper(self, side='leftright'):
         out *= planck(out.size, nleft=nleft, nright=nright)
         return out
 
-    def whiten(self, fftlength, overlap=0, method='scipy-welch',
-               window='hanning', detrend='constant', asd=None, **kwargs):
-        """White this `TimeSeries` against its own ASD
+    def whiten(self, fftlength=2, overlap=0, method='scipy-welch',
+               window='hanning', detrend='constant', asd=None,
+               fduration=1, highpass=None, **kwargs):
+        """Whiten this `TimeSeries` against its own ASD using inverse spectrum
+           truncation
```

Comment:
Can you shorten this docstring to

```
Whiten this `TimeSeries` using inverse spectrum truncation
```

? This will make it fit on one line, and the whitening doesn't have to use 'its own ASD'.

Reply:
This is fixed in commit 66e9982, along with a pesky split infinitive elsewhere in that doctoring.